cfn-resources/search-index/cmd/resource/resource.go

```diff
@@ -205,18 +206,31 @@ func newMappingsFields(fields []string) (map[string]interface{}, error) {
 
 	fieldsMap := make(map[string]interface{})
 	for _, p := range fields {
-		f := strings.Split(p, ":")
+		f := strings.SplitN(p, ":", 2)
```

Comment:
what was the reason for changing this?

Reply:
im only going to split in two parts, since the second one is a json may contain :